requirements.txt

```diff
@@ -6,11 +6,9 @@ opencv-python-headless==4.5.3.56
 scipy==1.6.0
 requests==2.25.1
 SQLAlchemy==1.3.22
-networkx<=3.1
---find-links https://download.pytorch.org/whl/torch_stable.html
-torch==1.8.1+cpu
-torchvision==0.9.1+cpu
-torchaudio===0.8.1
+torch
+torchvision
```

Comment:
can we as well rename this file as requirements_cpuonly.txt so its clear to folks what they're getting?

Reply:
Done